go/cmd/vttlstest/vttlstest.go

```diff
@@ -17,21 +17,16 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-	"fmt"
-	"io"
-
-	"github.com/spf13/pflag"
+	flag "github.com/spf13/pflag"
 
 	"vitess.io/vitess/go/exit"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
+	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/tlstest"
-
-	// Include deprecation warnings for soon-to-be-unsupported flag invocations.
-	_flag "vitess.io/vitess/go/internal/flag"
 )
 
+// nolint
```

Comment:
what was the linter error here?

Reply:
unused,deadcode. I'm going to move that content to a doc.go